cookiecutter.yaml

```diff
@@ -16,10 +16,10 @@ banner:
   location: Seattle, WA
   links:
   - url: intro.html
-    title: Event Jupyter Book
+    title: 2022 Jupyter Book
     new_window: false
-  - url: https://snowex.hackweek.io/intro.html
-    title: 2022 Website
+  - url: https://snowex-hackweek.github.io/website/tutorials/index.html
```

Comment:
Do you want it to go directly to the tutorials page?

Reply:
fixed